www/templates/intro/consent.html

```diff
@@ -1,113 +1,105 @@
 <ion-content>
 <div class="consent-title">
-<center><h4>E-Mission: Data driven carbon emission reduction</h4></center>
+<center><h4>ECO-ESCORT: Data driven carbon emission reduction</h4></center>
 </div>
 <div class="consent-text">
-<h3>Introduction and Purpose</h3>
-This app is part of a travel behavior research platform developed at the
-University of California, Berkeley. The platform development is being led by <a
-href="http://www.cs.berkeley.edu/~shankari">K. Shankari</a> under the
-supervision of <a href="http://www.cs.berkeley.edu/~culler/">Prof. David
-Culler</a>. Our goal is to collect information from users about travel behavior
-across all modes, and use the aggregate information for better transportation
-infrastructure planning.
+<h3>Introduction</h3>
+Our names are Andy Yan, Jack Sullivan, Jesse Zhang, Kyle Tse, Simon Behar, John Leyden and
```

Comment:
Fix Simon's name

Reply:
Not sure what you mean by this

Reply:
Simon is no longer part of the project, no? But I guess we can't actually change this because the consent document has to be unchanged. Bummer.